src/screens/EncounterHomeScreen.tsx

```diff
@@ -20,16 +21,27 @@ import {
 } from '../components';
 import { LoadingSpinner } from '../components/LoadingSpinner';
 import { colors, consts, device, Icon, normalize, texts } from '../config';
-import { useEncounterPolling, useEncounterUser, useQRValue } from '../hooks';
+import {
+  useEncounterPolling,
+  useEncounterUser,
+  useOpenWebScreen,
+  useQRValue,
+  useStaticContent
+} from '../hooks';
 import { QUERY_TYPES } from '../queries';
+import { SettingsContext } from '../SettingsProvider';
 import { ScreenName } from '../types';
 
 const INFO_ICON_SIZE = normalize(16);
 
 const a11yLabels = consts.a11yLabel;
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
+// eslint-disable-next-line complexity
 export const EncounterHomeScreen = ({ navigation, route }: any) => {
+  // @ts-expect-error settings are not properly typed
+  const encounterCategoryId = useContext(SettingsContext).globalSettings.settings?.encounter
+    ?.categoryId;
```

Comment:
so we need to add a new settings config on the main server with the value of the encounter category, right?

Reply:
correct. i added it to the documentation in confluence